Dockerfile

```diff
@@ -1,8 +1,6 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM python:3.8-slim-buster
```

Comment:
readme mentions `Python 3.9.x installed`

Reply:
Also, before merging this, it'd be nice to check if it'll actually work, as just running `python3 -m pip install -r requirements.txt` might not be enough to get the bot up and running on a raspberry pi, etc...

Reply:
Wow, sorry about the version.... I'm testing it on a Raspberry PI 3. I'll let you know here when 3.9 is tested.

Reply:
How does this effect the normal Docker image that @letsautom8 setup? Does this change the architecture for the existing Docker image?

@letsautom8, are you in the Telegram chat with us?

Reply:
I am not in telegram chat. Invite me @letsautom8

Reply:
Just added you. They are updating your Dockerfile to work with arm/v7 on a RPi. I just want to make sure this change does not break what you implemented before I merge it.

Reply:
Hi. This change should not break any linux/amd64 because python:3.9 base images supports it. The missing part I'm trying to test is to publish a muti-arch image in a test repository and share with you the build steps to have it automated. Are you using Github workflows?

Reply:
@letsautom8, set up the initial Docker setup and Github workflows. I just wanted to confirm with him that this update does not break what he did.

Reply:
I never used Github workflows, but looking at the container.yml it looks very clear and I think should work. For me its, working with this change, but please, test it locally. One thing you could do is to mantain two Dockerfiles add a tag suffix to make the migration more safe. Very nice work guys!

Reply:
I suggest creating a separate Dockerfile for raspberry pi, name it Dockerfile.arm64 or something

Reply:
When this update has been done and pushed let me know.

Reply:
@plitex, are you going to update the Docker files and push the update?

Reply:
Ok, will do. Thanks.